projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -70,6 +75,43 @@ export default function PublicizeForm( {
 		[ autosave, isEditedPostDirty ]
 	);
 
+	const renderNotices = () => (
+		<>
+			{ brokenConnections.length > 0 && (
+				<Notice type={ 'error' }>
+					{ createInterpolateElement(
+						_n(
+							'One of your social connections is broken. Reconnect them on the <fixLink>connection management</fixLink> page.',
+							'Some of your social connections are broken. Reconnect them on the <fixLink>connection management</fixLink> page.',
+							brokenConnections.length,
+							'jetpack'
+						),
+						{
+							fixLink: <ExternalLink href={ connectionsAdminUrl } />,
+						}
+					) }
+				</Notice>
+			) }
+			{ unsupportedConnections.length > 0 && (
+				<Notice type={ 'error' }>
+					{ createInterpolateElement(
+						_n(
+							'One of your social connection is not supported anymore. <moreInfo>Learn more here</moreInfo>.',
+							'Some of your social connections are not supported anymore. <moreInfo>Learn more here</moreInfo>.',
+							unsupportedConnections.length,
+							'jetpack'
+						),
+						{
+							moreInfo: (
+								<ExternalLink href={ getRedirectUrl( 'publicize-connection-not-supported' ) } />
```

Comment:
It's seems that we can only have different redirect URLs for Jetpack and WPCOM if we include the site ID/slug in the target URL!

We could fake it and and the site identifier in a query parameter, even though it won't be used, make a change to Jetpack redirects, or pass in a different URL in the initial state, like we do with the `connection-test-results` URL. The last might be the quickest and simplest for now.